### PR TITLE
sync sdk

### PR DESCRIPTION
**Pull request by dineshyv**

### CODE REVIEW

File: src/llama_stack_client/types/shared/photogen_tool_definition.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from typing_extensions import Literal
 
 from ..._models import BaseModel
-from ..rest_api_execution_config_param import RestAPIExecutionConfigParam
+from ..rest_api_execution_config import RestAPIExecutionConfig
```
Comment (yanxi0830) [inline src/llama_stack_client/types/shared/photogen_tool_definition.py:7]: I think this will produce an error. Does `llama-stack-client` work?

Comment (yanxi0830) [reply src/llama_stack_client/types/shared/photogen_tool_definition.py:7]: Let me add to the sync script to fix this

Comment (dineshyv) [reply src/llama_stack_client/types/shared/photogen_tool_definition.py]: fixed.

**Comment by dineshyv** [comment]
merging to unblock

**Pull request merged**